app/models/calculated_handover_date.rb

```diff
@@ -5,6 +5,10 @@ class CalculatedHandoverDate < ApplicationRecord
   CUSTODY_WITH_COM = 'CustodyWithCom'
   COMMUNITY_RESPONSIBLE = 'Community'
 
+  COM_NO_HANDOVER_DATE = new responsibility: COMMUNITY_RESPONSIBLE,
+                             start_date: nil, handover_date: nil,
+                             reason: :com_responsibility
```

Comment:
You are correct and moving this to CalculatedHandoverDate was a bit of an after thought which didnt feel quite right. OffenderHandover is actually a great place to put it.

Thank you for throwing up this feedback rather than a LGTM!